packages/react/src/useTableUtils/types.ts

```diff
@@ -36,6 +36,7 @@ export interface TableOptions {
   initialCursor?: string;
   initialDirection?: "forward" | "backward";
   columns?: (string | RelatedFieldColumn | CustomCellColumn)[];
+  exclude?: string[];
```

Comment:
Could you rename this to `excludeColumns`

I'm adding a system that does a similar `include` thing with actions, and I think that it would be nice to have `actions` & `excludeActions` with names that align with the columns